### PR TITLE
WAI 16 Gross Enrolment Ration Map Overlays

### PR DESCRIPTION
**Pull request by tcaiger**
### Changes:

- add mapOverlay utils for migrations
- add LESMIS GER map overlays

---

### Screenshots:
See issue

### CODE REVIEW

File: packages/database/src/utilities/mapOverlays.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+export const getMapOverlayGroupId = async (db, code) => {
```
Comment (EMcQ-BES) [inline packages/database/src/utilities/mapOverlays.js:6]: There's a function under `database/src/utilities/migration.js` called `codeToId` which does this generically across any table

File: packages/database/src/utilities/mapOverlays.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  const results = await db.runSql(`select "id" from "map_overlay_group" where "code" = '${code}';`);
+
+  if (results.rows.length > 0) {
+    return results.rows[0].id;
+  }
+
+  throw new Error('MapOverlayGroup not found');
+};
+
+export const getPermissionGroupId = async (db, name) => {
```
Comment (EMcQ-BES) [inline packages/database/src/utilities/mapOverlays.js:16]: I think this exact function has been declared individually in about 20 different migrations, so probably a good call to split it out to a util function, but I wouldn't expect to find this in a file called `mapOverlays` 🤔 

I reckon get rid of the mapOverlay function for the generic codeToId and then rename this file to match this util function

**Review by EMcQ-BES — CHANGES_REQUESTED** [review]
The main body of the migration looks good, just a couple notes on the utils you added.

**Review by EMcQ-BES — APPROVED** [review]

File: packages/database/src/utilities/mapOverlays.js (hunk 1 above)
Comment (tcaiger) [reply packages/database/src/utilities/mapOverlays.js]: Thanks for that. It's a handy util to be aware of.

**Pull request merged**